travis/install_grpc.sh

```diff
@@ -13,6 +13,7 @@ fi
 
 git clone https://github.com/grpc/grpc.git
 cd grpc
+git reset --hard fa507530590944683a0c672fcd4ea5a54977c903
```

Comment:
Yeah, I do too, I think I changed it because something wasn't working and I wanted to be really sure it wasn't the git checkout...

Anyways, will change and merge.